### PR TITLE
Implemented code to take instanceUrl from user and use it (#1wf8d38)

### PR DESCRIPTION
**Pull request by bashu22tiwari**
### Related Issues

Closes #

### Screenshots of Visual Changes before/after (If There Are Any)

**IMPORTANT NOTICE** - Remember to update `CHANGELOG.md` with description of your change

### Contribution and Currently Important Rules Acceptance

- [ ] I read and followed [contribution rules](https://github.com/DivanteLtd/vsf-capybara/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: changelogs/unreleased/-1wf8d38.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+title: Implemented code to take instanceUrl from user and use it (#1wf8d38)
```
Comment (ymaheshwari1) [inline changelogs/unreleased/-1wf8d38.yml:2]: Removed id from the title.

File: src/store/modules/user/mutations.ts [hunk 2]
```diff
@@ -16,6 +16,9 @@ const mutations: MutationTree <UserState> = {
     },
     [types.USER_CURRENT_FACILITY_UPDATED] (state, payload) {
         state.currentFacility = payload;
+    },
+    [types.USER_INSTANCE_URL_UPDATED] (STATE, payload) {
+        STATE.instanceUrl = payload;
```
Comment (ymaheshwari1) [inline src/store/modules/user/mutations.ts:21]: Change `STATE` to `state`

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: src/store/modules/user/mutations.ts (hunk 2 above)
Comment (bashu22tiwari) [reply src/store/modules/user/mutations.ts]: Changed.
Thank You for reviewing my code.

File: changelogs/unreleased/-1wf8d38.yml (hunk 1 above)
Comment (bashu22tiwari) [reply changelogs/unreleased/-1wf8d38.yml]: Removed.
Thank You for reviewing my code.

**Review by ymaheshwari1 — APPROVED** [review]

File: src/views/Login.vue [hunk 3]
```diff
@@ -48,11 +53,21 @@ export default defineComponent({
...
     return {
       username: "",
-      password: ""
+      password: "",
+      instanceUrl: ""
     };
   },
+  computed: {
+    ...mapGetters({
+      currentInstanceUrlSaved: 'user/getInstanceUrl'
```
Comment (adityasharma7) [inline src/views/Login.vue:62]:
```suggestion
      currentInstanceUrl: 'user/getInstanceUrl'
```

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

File: src/views/Login.vue (hunk 3 above)
Comment (bashu22tiwari) [reply src/views/Login.vue]: Changed the name.
Thank You Sir for reviewing my code.

**Review by adityasharma7 — APPROVED** [review]

**Pull request merged**